penguins/tf-keras/trainer/util.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities to download and preprocess the Penguin data."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+from six.moves import urllib
+import tempfile
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+# Keras specific
+from tensorflow.keras.utils import to_categorical
+
+# Storage directory
+DATA_DIR = os.path.join(tempfile.gettempdir(), "penguins_data")
+
+# Download options.
+DATA_URL = "https://storage.googleapis.com/cloud-samples-data/ai-platform/penguins"
+TRAINING_FILE = "penguins.data.csv"
```

Comment:
small nit: is that a typical filepath?

I've usually seen stuff like `penguins_train.csv`

Reply:
@aribray Please remove gs://cloud-samples-data/ai-platform/penguins/penguins.csv if you're not using it anymore.